File: quizlet_repository/src/main/java/com/example/alexeyglushkov/quizletservice/QuizletCommandProvider.java
package com.example.alexeyglushkov.quizletservice;

import com.example.alexeyglushkov.authorization.Auth.ServiceCommandProvider;
import com.example.alexeyglushkov.cachemanager.clients.Cache;
import com.example.alexeyglushkov.streamlib.progress.ProgressListener;

/**
 * Created by alexeyglushkov on 03.04.16.
 */
public interface QuizletCommandProvider extends ServiceCommandProvider {
    QuizletSetsCommand getLoadSetsCommand(String server, String userId, ProgressListener progressListener);
}

File: quizlet_repository/src/main/java/com/example/alexeyglushkov/quizletservice/QuizletCommandProvider.kt
package com.example.alexeyglushkov.quizletservice

import com.example.alexeyglushkov.authorization.Auth.ServiceCommandProvider
import com.example.alexeyglushkov.streamlib.progress.ProgressListener

/**
 * Created by alexeyglushkov on 03.04.16.
 */
interface QuizletCommandProvider : ServiceCommandProvider {
    fun getLoadSetsCommand(server: String, userId: String, progressListener: ProgressListener): QuizletSetsCommand
}